extensions/amp-auto-ads/0.1/test/test-alright-network-config.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Services} from '../../../../src/services';
+import {getAdNetworkConfig} from '../ad-network-config';
+
+describes.realWin(
+  'alright-network-config',
+  {
+    amp: {
+      canonicalUrl: 'https://foo.bar/baz',
+      runtimeOn: true,
+      ampdoc: 'single',
+    },
+  },
+  (env) => {
+    let ampAutoAdsElem;
+    let document;
+
+    beforeEach(() => {
+      document = env.win.document;
+      ampAutoAdsElem = document.createElement('amp-auto-ads');
+      env.win.document.body.appendChild(ampAutoAdsElem);
+    });
+
+    afterEach(() => {
+      env.win.document.body.removeChild(ampAutoAdsElem);
+    });
+
+    describe('Alright', () => {
+      const PUBLISHER_ID = '1';
+      const PAGE_TYPE = 'home';
+
+      beforeEach(() => {
+        ampAutoAdsElem.setAttribute('data-publisher-id', PUBLISHER_ID);
+        ampAutoAdsElem.setAttribute('data-page-type', PAGE_TYPE);
+      });
+
+      it('should report enabled always', () => {
+        const adNetwork = getAdNetworkConfig('alright', ampAutoAdsElem);
+        expect(adNetwork.isEnabled(env.win)).to.equal(true);
+      });
+
+      it('should generate the config fetch URL', () => {
+        const adNetwork = getAdNetworkConfig('alright', ampAutoAdsElem);
+        const configUrl = adNetwork.getConfigUrl();
+
+        expect(configUrl).to.contain('//analytics.alright.network/amp/');
+        expect(configUrl).to.contain('p=' + PUBLISHER_ID);
+        expect(configUrl).to.contain('t=' + PAGE_TYPE);
+        expect(configUrl).to.contain('u=https%3A%2F%2Ffoo.bar%2Fbaz');
+      });
+
+      it.skip("should truncate the URL if it's too long", () => {
```

Comment:
why are we adding a skipped test?

Reply:
This test isn't necessary, I will remove it.